package.json

```diff
@@ -34,11 +35,12 @@
     "@babel/core": "7.1.5",
     "@babel/preset-env": "7.1.5",
     "@babel/register": "7.0.0",
+    "@types/node": "^11.11.3",
```

Comment:
:lock: Prefer locked versions over fuzzy versions. Instead of `^11.11.3`, should be `11.11.3`.